src/contracts/LidoLiquidityManager.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {OwnableOperable} from "./OwnableOperable.sol";
+import {IERC20, IWETH, IStETHWithdrawal} from "./Interfaces.sol";
+
+/**
+ * @title Manages stETH liquidity against the Lido Withdrawal Queue.
+ * @author Origin Protocol Inc
+ */
+abstract contract LidoLiquidityManager is OwnableOperable {
+    IERC20 public immutable steth;
+    IWETH public immutable weth;
+    IStETHWithdrawal public immutable withdrawalQueue;
+
+    uint256 public outstandingEther;
+
+    uint256[49] private _gap;
+
+    constructor(address _steth, address _weth, address _lidoWithdrawalQueue) {
+        steth = IERC20(_steth);
+        weth = IWETH(_weth);
+        withdrawalQueue = IStETHWithdrawal(_lidoWithdrawalQueue);
+    }
+
+    /**
+     * @dev Approve the stETH withdrawal contract. Used for redemption requests.
+     */
+    function _initLidoLiquidityManager() internal {
+        steth.approve(address(withdrawalQueue), type(uint256).max);
+    }
+
+    /**
+     * @notice Approve the stETH withdrawal contract. Used for redemption requests.
+     */
+    function approveStETH() external onlyOperatorOrOwner {
```

Comment:
🟠 This is redundant and can be removed? Since we will be initing the contract, and that init will call `_initLidoLiquidityManager()`?

Reply:
yes, I will remove